src/test_all.py

```diff
@@ -140,7 +157,14 @@ def assert_language_name(name):
                     except KeyError:
                         raise Exception(f" * Looks like we have a repeated example here! "
                                         f"Merge outputs before removing duplicates. :-/ \n {instance}")
-
+                        
+                        
+            #Make sure classes are balanced
+            output=[ins['output'] for ins in instances]
+            outputs = sum(output, [])
```

Comment:
What is the purpose of this line?

Reply:
Each `ins['output']` is a list, so I get the real outputs in the following line.

Reply:
Got it. Maybe add a comment just above the code that indicates that this line "flattens the nested arrays"